docs/source/learn/core_notebooks/dims_module.ipynb

```diff
@@ -0,0 +1,2910 @@
+{
```

Comment:
<p>Is <code>pymc.dims.as_xtensor</code>correct? I find it surprising to use <code>as_xtensor</code> to get a regular non-dimmed variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/7820/discussion/#comment-2200857928'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>